src/agent/debuglet.ts

```diff
@@ -368,6 +365,15 @@ export class Debuglet extends EventEmitter {
 
     });
   }
+
+  /**
+   * isReady is designed to support debug agent on Google Cloud Function (GCF).
```

Comment:
Nit: The user doesn't know about CachedPromise (nor should they) -- to them, they should call this function and expect to get a Promise that's resolved when they know that the list of breakpoints is no longer stale. Can you re-frame this explanation to account for this? Keep in mind that this is on the public API of the Debug Agent now.

Reply:
Done.